src/common/Pagination/index.js

```diff
@@ -1,34 +1,52 @@
-import { Button, PageChanger, PageCounter, Text } from "./styled";
-import { ReactComponent as Previous } from "./previousArrow.svg";
-import { ReactComponent as Next } from "./nextArrow.svg";
+import {
+  ArrowIcon,
+  ArrowIconNext,
```

Comment:
Ja jednak trzymalbym, sie tego, ze jak jedna strzalka ma w nazwie next to, zeby dodac previous do tej drugiej, bo inaczej jest to mniej czytelne :)